.github/workflows/ci.yml

```diff
@@ -1,35 +1,37 @@
-name: Test this action
+name: CI
```

Comment:
If someone uses this action with other actions, a semantic name will be better.

```suggestion
name: SUI Lighthouse CI
```

Reply:
But, this is the workflow name for this repo, not the shared action